tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_additional.robot

```diff
@@ -31,15 +39,67 @@ Verify Notebook Tolerations Are Applied To Workbenches When Set Up
     Set Pod Toleration Via UI    ${TOLERATIONS}
     Save Changes In Cluster Settings
     Launch Data Science Project Main Page
-    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
     Open Data Science Project Details Page       project_title=${PRJ_TITLE}
-    Sleep   5s
+    Sleep   20s
```

Comment:
`Open Data Science Project Details Page` already has some `Wait Until ...` Is this Sleep necessary?  If yes, maybe it could be added to the previous keyword

Reply:
Open Data Science Project Details Page is used also in other situation where a such sleep is not needed.
The 20s sleep is necessary to give enough time to the Dashboard to capture the new settings. I should have set a `reason`

Reply:
done